2025-02-26 09:58:47.886 [info] Experiment 'pythonaa' is active
2025-02-26 09:58:47.886 [info] Experiment 'pythonRecommendTensorboardExt' is active
2025-02-26 09:58:47.886 [info] Experiment 'pythonTerminalEnvVarActivation' is active
2025-02-26 09:58:47.886 [info] Experiment 'pythonTestAdapter' is active
2025-02-26 09:58:47.886 [info] Native locator: Refresh started
2025-02-26 09:58:47.886 [info] Native locator: Refresh started
2025-02-26 09:58:47.886 [info] > pyenv which python
2025-02-26 09:58:47.886 [info] cwd: .
2025-02-26 09:58:47.949 [info] Python interpreter path: ./odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python
2025-02-26 09:58:48.215 [error] Failed to start language server: Launching server using command /home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python failed. Error: spawn /home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python ENOENT
2025-02-26 09:58:48.223 [error] [
  'Failed to start language server, Class name = l, completed in 239ms, has a falsy return value, Arg 1: <Uri:/home/anhken>, Arg 2: {"id":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","sysPrefix":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv","envType":"Unknown","envName":"","envPath":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","path":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","architecture":3,"sysVersion":"","version":{"raw":"3.10.12","major":3,"minor":10,"patch":12,"build":[],"prerelease":[]},"displayName":"Python 3.10.12","detailedDisplayName":"Python 3.10.12"}, Arg 3: {"documentSelector":[{"scheme":"file","language":"python"},{"scheme":"untitled","language":"python"},{"scheme":"vscode-notebook","language":"python"},{"scheme":"vscode-notebook-cell","language":"python"},{"scheme":"vscode-interactive-input","language":"python"}],"workspaceFolder":{"uri":{"$mid":1,"fsPath":"/home/anhken","external":"file:///home/anhken","path":"/home/anhken","scheme":"file"},"name":"anhken","index":0},"synchronize":{"configurationSection":["python"]},"outputChannel":{"name":"Python Language Server","logLevel":3},"revealOutputChannelOn":4,"initializationOptions":{"markupKindPreferred":"markdown","completion":{"resolveEagerly":false,"disableSnippets":true},"diagnostics":{"enable":true,"didOpen":true,"didSave":true,"didChange":true},"hover":{"disable":{"keyword":{"all":true}}},"workspace":{"extraPaths":["/home/anhken"],"environmentPath":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","symbols":{"maxSymbols":0}}},"middleware":{"serviceContainer":{"container":{"options":{"autoBindInjectable":false,"defaultScope":"Transient","skipBaseClassChecks":true},"id":0,"_bindingDictionary":{"_map":{}},"_snapshots":[],"_middleware":null,"_activations":{"_map":{}},"_deactivations":{"_map":{}},"parent":null,"_metadataReader":{},"_moduleActivationStore":{"_map":{}}}},"lastCaptured":{},"nextWindow":0,"eventCount":0,"workspace":{},"connectedPromise":{"scope":null,"_resolved":false,"_rejected":false,"_promise":{}},"notebooks":{},"eventName":"JEDI_LANGUAGE_SERVER.REQUEST"}}, Return Value: undefined',
  [Error: Launching Jedi language server using python failed, see output.
  	at l.start (/home/anhken/.vscode-server/extensions/ms-python.python-2025.0.0/out/client/extension.js:2:22183)
  	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
]
2025-02-26 09:58:48.227 [error] [
  'Starting language server, Class name = m, completed in 241ms, has a falsy return value, , Return Value: undefined',
  [Error: Launching Jedi language server using python failed, see output.
  	at l.start (/home/anhken/.vscode-server/extensions/ms-python.python-2025.0.0/out/client/extension.js:2:22183)
  	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
]
2025-02-26 09:58:48.228 [error] [
  'Failed to start language server, Class name = m, completed in 280ms, has a falsy return value, Arg 1: <Uri:/home/anhken>, Arg 2: {"id":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","sysPrefix":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv","envType":"Unknown","envName":"","envPath":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","path":"/home/anhken/odoo-fitdnu/odoo-fitdnu/odoo-fitdnu/venv/bin/python","architecture":3,"sysVersion":"","version":{"raw":"3.10.12","major":3,"minor":10,"patch":12,"build":[],"prerelease":[]},"displayName":"Python 3.10.12","detailedDisplayName":"Python 3.10.12"}, Return Value: undefined',
  [Error: Launching Jedi language server using python failed, see output.
  	at l.start (/home/anhken/.vscode-server/extensions/ms-python.python-2025.0.0/out/client/extension.js:2:22183)
  	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
]
2025-02-26 09:58:48.229 [error] [
  'Failed to activate a workspace, Class name = v, completed in 1189ms, has a falsy return value, Arg 1: <Uri:/home/anhken/odoo-fitdnu/addons/quan_ly_van_ban/__manifest__.py>, Arg 2: {"started":1740538726176}, Return Value: undefined',
  [Error: Launching Jedi language server using python failed, see output.
  	at l.start (/home/anhken/.vscode-server/extensions/ms-python.python-2025.0.0/out/client/extension.js:2:22183)
  	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
]
2025-02-26 09:58:48.229 [error] Failure during activation. [Error: Launching Jedi language server using python failed, see output.
	at l.start (/home/anhken/.vscode-server/extensions/ms-python.python-2025.0.0/out/client/extension.js:2:22183)
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
2025-02-26 09:58:48.229 [error] sendStartupTelemetry() failed. [Error: Launching Jedi language server using python failed, see output.
	at l.start (/home/anhken/.vscode-server/extensions/ms-python.python-2025.0.0/out/client/extension.js:2:22183)
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
2025-02-26 09:58:55.675 [info] Native locator: Refresh finished in 9310 ms
